_posts/2016-12-04-contributing-blog-post-to-pulpproject.org.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Contributing a blog post to pulpproject.org
+layout: post
```

Comment:
This is not needed and can be pulled from this post and the recommendation. Anything in _posts defaults to this layout.